README.md

```diff
@@ -4,55 +4,146 @@
+
+### Python
+
+On the Rocky 9 machine, install development tools for Python:
+
 ```bash
 sudo dnf install "@Development Tools" python3.11 python3.11-pip python3.11-setuptools python3.11-devel openldap-devel git
 ```
 
-[Poetry](https://python-poetry.org/) is used to manage the dependencies of this API. To install Poetry, follow [the instructions](https://python-poetry.org/docs/) from their documentation.
+Clone the codebase
 
-To install the project's dependencies, execute `poetry install`. The dependencies and the code in this repo are compatible with Python >=3.11, <3.12.
+```bash
+git clone https://github.com/ral-facilities/operationsgateway-api.git
+```
+
+### Poetry
 
-## Nox Sessions
-Like [DataGateway API](https://github.com/ral-facilities/datagateway-api), this repository contains a [Nox](https://nox.thea.codes) file (`noxfile.py`) which exists in the root level of this repository. There are a handful of sessions which help with repetitive tasks during development To install Nox, use the following command:
+[Poetry](https://python-poetry.org/) is used to manage the dependencies of this API.
 
 ```bash
-python3.11 -m pip install --user --upgrade nox
+# Install poetry
+curl -sSL https://install.python-poetry.org | python3 -
+# Change the directory to where the poetry .toml file is
+cd operationsgateway-api
+# Set poetry to use python 3.11
+poetry env use python3.11
+# Install the dependencies
+poetry install
 ```
 
-To run a specific Nox session, use the following:
+### MongoDB:
+
+Create the following file:
 
 ```bash
-nox -s [SESSION NAME]
+sudo vi /etc/yum.repos.d/mongodb-org-7.0.repo
 ```
 
-Currently, the following Nox sessions have been created:
-- `black` - this uses [Black](https://black.readthedocs.io/en/stable/) to format Python code to a pre-defined style.
-- `lint` - this uses [flake8](https://flake8.pycqa.org/en/latest/) with a number of additional plugins (see the included `noxfile.py` to see which plugins are used) to lint the code to keep it Pythonic. `.flake8` configures `flake8` and the plugins.
-- `safety` - this uses [safety](https://github.com/pyupio/safety) to check the dependencies (pulled directly from Poetry) for any known vulnerabilities. This session gives the output in a full ASCII style report.
-- `tests` - this uses [pytest](https://docs.pytest.org/en/stable/) to execute the automated tests in `test/`. There are currently no automated tests written for this repository however.
+Add:
 
-## API Configuration
-In `operationsgateway_api/`, there is an example config file (`config.yml.example`). Copy this example so `config.yml` exists in the same directory level and edit the configuration as needed for your system. There is also a logging configuration file (`logging.ini.example`). Follow the same procedure as the config file as to set up the logging.
+```bash
+[mongodb-org-7.0]
+name=MongoDB Repository
+baseurl=https://repo.mongodb.org/yum/redhat/8/mongodb-org/7.0/x86_64/
```

Comment:
Given the instructions are intended to be tailored for Rocky9, should this use https://repo.mongodb.org/yum/redhat/9/ instead of https://repo.mongodb.org/yum/redhat/8/? My dev VM is Rocky8 so mine does match this but I'd have thought it would be different to work on 9?

Reply:
They must be cross-compatible as I've been using redhat/8 for a while, however, updating to 9 works fine as well. I have updated the readme.